cupy/core/raw.pxd

```diff
@@ -6,6 +6,7 @@ cdef class RawKernel:
         readonly tuple options
         object _kernel
         readonly str backend
+        bint cuComplex
```

Comment:
nitpick: use `translate_cucomplex` for consistency and PEP8

Reply:
done